src/travertino/node.py

```diff
@@ -108,7 +99,21 @@ def remove(self, child):
 
         self._children.remove(child)
         child._parent = None
-        set_root(child, None)
+        self._set_root(child, None)
+
+    def clear(self):
+        """Clear all children from this node.
+
+        Raises:
+            ValueError: If this node is a leaf, and cannot have children.
+        """
+        if self._children is None:
+            raise ValueError('Cannot remove children')
```

Comment:
Why raise an error here, rather than just "return after doing nothing?" If it's silent, any iterative usage of the method can just call it and know the job is done; if it raises an error, you'll have to catch the error (and, in all cases I can think of, do nothing). Have you got a specific case in mind where calling `clear` on a leaf should raise an error?